docs/4_3_co-simulation_schema.adoc

```diff
@@ -32,21 +35,6 @@ _[Typically, this information is only utilized for information purposes.
 For example, a co-simulation master can inform the user that a tool has to be available on the computer where the slave is instantiated.
 The name of the tool can be taken from attribute `generationTool` of `fmiModelDescription`.]_
 
-|`canHandleVariableCommunicationStepSize`
-|The slave can handle variable communication step size.
-The communication step size (parameter communicationStepSize of `fmi3DoStep`) has not to be constant for each call.
-
-|`canInterpolateInputs`
```

Comment:
This seems to be deleted on error (probably by me!?). `canInterpolateInputs` is currently under discussion if it should be replaced by `maxInputDerivativeOrder` but should currently still be part of the documentation.